src/main/java/com/ubc/cpsc304/hotelify/HotelifyConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package com.ubc.cpsc304.hotelify;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+/**
+ * Customized Configurations.
+ * Created by ao on 2018-11-04
+ */
+@Configuration
+public class HotelifyConfiguration implements WebMvcConfigurer {
```

Comment:
should be named as `HotelifyWebMvcConfiguration`, in case we need an actual `HotelifyConfiguration` file in the future

Reply:
cool